sample-app/src/main/java/com/sinch/sample/conversation/messages/Send.java

```diff
@@ -119,5 +126,8 @@ SendMessageRequest<?> createSMSSendMessage() {
                     .build()))
         .setTtl(25)
         .build();
+
+    webhooksConversationPath.ifPresent(builder::setCallbackUrl);
```

Comment:
Suggestion: wouldn't is be clearer to use the setter in the builder chain above?
```
.setCallbackUrl(webhooksConversationPath.orElse(null));
```

Reply:
no: this will lead to send the null value as the `callback_url` value
Setting a value (even null) mean: "send the value"

Reply:
Arf, that's true. So you are forced to do this manipulation to use an optional parameter in the builder whose value may come or be empty from the configuration file 😣

Reply:
Yes. By purpose.
We are in samples here: if configuration file define a value it supersed the default from dashboard (if any)
If not set, not sent and use dashboard configuration
Again: just a matter of samples; SDK itself is not in the loop